frontend/src/styling/StyledHome.ts

```diff
@@ -1,5 +1,6 @@
 import styled from 'styled-components';
-import { colors } from '../constants/Colors';
+import { colors } from '../constants/colors';
```

Comment:
We can import from '../constants' because the folder has an index file

Reply:
Updated